usdt-impl/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 
 #![cfg_attr(usdt_need_feat_asm, feature(asm))]
 #![cfg_attr(usdt_need_feat_asm_sym, feature(asm_sym))]
+#![cfg_attr(usdt_need_feat_used_with_arg, feature(used_with_arg))]
```

Comment:
Hmm. I don't know if I realized it before, but this feature is not stable. We've recently been able to support a stable compiler on all platforms, and it would be very nice to keep that. I'm not sure that's possible, so FreeBSD might be the hold-out requiring nightly.

Are you certain that the `link-dead-code` linker-flag is not sufficient by itself to keep the FreeBSD linker from removing the start/stop symbols?

Reply:
With stable compiler and `link-dead-code`, it kinda works empirically under the limited cases I have tried but not that I can comfortably conclude it works (especially when it's failing tests on wrong mangled function name).

 Apart from the correctness issue, It also require consumer of the crate to add the linker flag to make it useful.
 
 😞